docs/en/operations/table_engines/mergetree.md

```diff
@@ -70,11 +70,13 @@ For a description of parameters, see the [CREATE query description](../../query_
 
     If a sampling expression is used, the primary key must contain it. Example: `SAMPLE BY intHash32(UserID) ORDER BY (CounterID, EventDate, intHash32(UserID))`.
 
-- `TTL` — An expression for setting storage time for rows.
+- `TTL` — A list of rules specifying durations of storage of rows and defining logic of automatic moving parts between disks and volumes.
 
-    It must have one `Date` or `DateTime` column as a result. Example:
+    Expression must have one `Date` or `DateTime` column as a result. Example:
     `TTL date + INTERVAL 1 DAY`
 
+    Type of the rule `DELETE|TO DISK 'xxx'|TO VOLUME 'xxx'` specified an action which is to be done with the part if expression is satisfied (reaches current time): deletion of expired rows, moving of part (if expression satisfies for all rows of a part) to specified disk (`TO DISK 'xxx'`) or volume (`TO VOLUME 'xxx'`). Default type of the rule is deletion (`DELETE`). Only one `DELETE` expression can be specified in the list of rules.
```

Comment:
```suggestion
    Type of the rule `DELETE|TO DISK 'xxx'|TO VOLUME 'xxx'` specifies an action to be done with the part if the expression is satisfied (reaches current time): removal of expired rows, moving a part (if expression is satisfied for all rows in a part) to specified disk (`TO DISK 'xxx'`) or to volume (`TO VOLUME 'xxx'`). Default type of the rule is deletion (`DELETE`). List of multiple rules can specified, but there should be no more than one `DELETE` rule.
```
probably needs a mention of what happens if all rows in a part satisfy multiple rules